modules/book_handler.js

```diff
@@ -1,37 +1,41 @@
-import Books from './books.js';
+import Book from './books.js';
```

Comment:
### **Highlight**

Crowd clapping 👏 👏 for managing to separate your classes in this great way 💯